### PR TITLE
Calendar styling

### PR DESCRIPTION
**Pull request by JWLD**
Rel #71

### CODE REVIEW

File: public/css/main.css [hunk 1]
```diff
@@ -4,21 +4,27 @@ html {
...
   cursor: pointer;
 }
 
 a {
   text-decoration: none;
 }
 
+div, tr, td, span {
+  display: flex;
+  justify-content: center;
```
Comment (Karyum) [inline public/css/main.css]: are you sure you want everything to have display flex?
and what for ?

File: public/css/main.css [hunk 2]
```diff
@@ -34,3 +40,59 @@ body {
 /*** ADD EVENT FORM ***/
 
 /*** CALENDAR ***/
+
+.cal-wrap {
+  flex-direction: column;
+  position: fixed;
+  top: 0;
+  left: 0;
```
Comment (Karyum) [inline public/css/main.css]: is this doing anything?

File: public/js/calendar.js [hunk 3]
```diff
@@ -2,7 +2,7 @@ var Cal = function (divId) {
   this.divId = divId;
 
   // Days of week, starting on Sunday
-  this.DaysOfWeek = ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'];
+  this.DaysOfWeek = ['S', 'M', 'T', 'W', 'T', 'F', 'S'];
```
Comment (Karyum) [inline public/js/calendar.js]: I would say having the days 3 letters is clearer bit this looks better i guess.

File: public/css/main.css (hunk 1 above)
Comment (JWLD) [reply public/css/main.css]: Makes positioning everything a lot easier

File: public/css/main.css (hunk 2 above)
Comment (JWLD) [reply public/css/main.css]: Yea, position:fixed has different start positions on some browsers

File: public/js/calendar.js (hunk 3 above)
Comment (JWLD) [reply public/js/calendar.js]: Yea this was in the design that some legend made so

Comment (JWLD) [reply public/js/calendar.js]: But yea I don't think anyone would have any trouble working it out

File: public/css/main.css (hunk 2 above)
Comment (JWLD) [reply public/css/main.css]: Oh wait sorry I see your point, since it's width 100%. Yea fair

File: public/js/calendar.js (hunk 3 above)
Comment (Karyum) [reply public/js/calendar.js]: > Yea this was in the design that some legend made so
 LOL

File: public/css/main.css (hunk 2 above)
Comment (Karyum) [reply public/css/main.css]: So I don't understand are we going to get rid of it ?

Comment (JWLD) [reply public/css/main.css]: Nah just leave it, it doesn't matter and it's safer than not having it

**Pull request merged**